src/thematic/4_vegetation/sfp-401_vegopname-terrest_nl/07_stappenplan.Rmd

```diff
@@ -342,18 +332,14 @@ proefvlak. Vanuit dit hoekpunt kan dan gemakkelijk het volledige proefvlak
 worden gereconstrueerd met een kompas. Een duurdere maar met langere levensduur
 is het ingraven van een metalen staaf, die naderhand met een metaaldetector
 terug kan gezocht worden. Een andere oplossing is de proefvlakken te markeren
-met een fenopaal (Figuur \@ref(fig:image8+9)). Indien maaibeheer, fenopalen diep genoeg steken
+met een fenopaal (Figuur \@ref(fig:image8)). Indien maaibeheer, fenopalen diep genoeg steken
 (= maaiveld) door een zode uit te graven, alvorens de fenopaal te plaatsen. Een
 metaaldetector is nadien wel handig.
 
-![](./media/image8.jpeg){width="45%"}
-![](./media/image9.jpeg){width="50%"}
 
 
-```{r image8+9, fig.cap = 'Een fenopaal, die gebruikt wordt om de punten te markeren die met GPS
-werden gepositioneerd (foto: Nathalie Cools (links), Geert Sioen (rechts))', out.width= '100%' }
-include_graphics(path = './media/image8.jpeg',
-include_graphics(path = './media/image9.jpeg')
+```{r image8, fig.cap = 'Een fenopaal, die gebruikt wordt om de punten te markeren die met GPS werden gepositioneerd (foto: Nathalie Cools (links), Geert Sioen (rechts))', out.width= '50%' }
+knitr::include_graphics(c((path = './media/image8.jpeg'),(path = './media/image9.jpeg')))
```

Comment:
```suggestion
knitr::include_graphics(path = c('./media/image8.jpeg',  './media/image9.jpeg'))
```

Reply:
Can I accept this suggested change somewhere (is that the add suggestion to batch button?) Or do I have to do this manually in my branch? 
(I already did the git pull origin master and now want to continue to adapt the sfp401_nl)

Reply:
@ElsDeBie see https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request
Accepting suggested changes here will add them as commit(s) in the sfp-401 branch of the remote (not yet on your local branch), which you can then pull locally (e.g. in order to subsequently add further commits locally and then push those again to the remote).

Reply:
Considering the feedback of @hansvancalster :
 > So it is best that you do `git pull origin master` with this branch checked out.

I guess it could be done both before or after accepting the suggestions here.

As you already merged master before accepting the suggestions, you will **first have to push** that update of sfp-401 to the remote (here) before accepting the suggestions here. Otherwise, you will have two diverging histories of sfp-401 locally vs. on the remote.